NuGet.Docs/Release-Notes/3.5-RC.md

```diff
@@ -0,0 +1,94 @@
+#3.5 RC Release Notes
+
+[Full Changelog]()
```

Comment:
Suggestion: Add a short summary: something like in this release we focused on bug fixes and added a 4 features.